DANS-KNAW/dd-ingest-flow
# DD-1227 make dansBagValidatorClient part of validateDansBag configura…

# Description of changes
- `dansBagValidatorClient` is renamed to `httpClient` and it is now part of `validateDansBag
` block in config.yml file.
- Code changes to get and set `httpClient` as  `dansBagValidatorClient`

# How to test
Suggestion: Run the service with `check` option
    ===> `INFO  [2023-01-26 11:41:01,751] [main] io.dropwizard.cli.CheckCommand: Configuration is OK`
# Related PRs

(Add links)

*

# Notify

@DANS-KNAW/dataversedans
